test/integration/targets/win_say/tasks/main.yml

```diff
@@ -1,18 +1,24 @@
+# CI hosts don't have a valid Speech package so we rely on check mode for basic
+# sanity tests
```

Comment:
Is it practical to install a speech package as part of the test? How much run time would it add to the test?

Reply:
I think it's more missing the actual audio capabilities to say the text when running on CI. Considering we have no way of verifying that it was said and all check mode does right now is skip the actual speech part I'm comfortable having it just cover this for now.